### ISSUE TITLE
"crowdfunding" CZ/SK modulu

### ISSUE THREAD

**Issue by radimklaska** [Drupalcz/community#5]
Jak nastavit model financovani?
* ma to to czda proste zaplatit a cekat jestli nekdo "podekuje" darem?
* nejdrive cekat na klienty, vybrat startovni castku? pak bod 1
* mely by se slozit firmy?

Jake moduly?

treti strana | Kategorie | pocet klientu | pocet firem | vyvojar | integrace
------------ | ------------- | ---------------- | --------------- | --------- | -------------
https://www.gopay.cz | platebni brana | 4 | 0 | ? | https://doc.gopay.com
http://www.payu.cz/ | platebni brana | 1 | 0 | ? | https://www.payu.pl/en/developers , https://github.com/PayU/openpayu_php
https://www.thepay.cz/ | platebni brana | 1 | 0 | ? | https://www.thepay.cz/ke-stazeni/
https://www.dpd.com/cz | prepravni sluzba | 0 | 0 | ? | http://pickup.dpd.cz/integrace/
https://www.ceskaposta.cz | prepravni sluzba | 1 | 0 | ? | https://www.ceskaposta.cz/uzitecne-nastroje/navody-pro-e-shopy#2
https://www.balikobot.cz/ |  api k prepravnim sluzbam | 0 | 0 | ? | ?
https://www.mojeid.cz/ | autentifikace | 0 | 0 | ? | D8 port https://www.drupal.org/project/mojeid

Kdo z programatoru ma zajem o takovou praci? Firma? Cast by se mohla udelat formou sprintu.

**Comment by landsman** [comment]
@radimklaska mojeID fakt nekdo ještě používá?

Osobně jsem pro programování **GoPay**, rád přiložím ruku k dílu.

Jakým způsobem by se tenhle modul dělal? Pro modul Commerce jako gateway nebo nějak univerzálně s podporou různých e-commerce řešení / webforms, ... ?

K financování: Vyhlásil bych, že chceme tenhle modul vyrobit a že bychom rádi vybrali nějaké peníze aka právě "crowdfunding". Nějaké platby ex-post bych si moc nedělal iluze.

**Comment by martin-klima** [comment]
Rád pomohu s GoPay, případně dalšími vychytávkami do commerce pro CZ prostředí.

**Comment by miloskroulik** [comment]
@landsman Jo, používá. Např. náš současn klient, ale viděl jsem to i jinde na českých webech poměrně často, proto jsem to navrhl. Ale je možné, že to umírá...

**Comment by secmi** [comment]
Osobně jsem také za formu crowdfunding, pro začátkem s nějakým opravdu žádaným modulem.

Nejtěžší asi bude to celé zpropagovat, přesvědčit lidi aby zaplatili a hlavně jim nabídnout nějakou výhodu, protože předpokládám, že časem ten modul stejně půjde ven.

**Comment by radimklaska** [comment]
@landsman MojeID prislo jako navrh... Pokud o to je skutecne zajem, tak proc ne - o tom uz by rozhodla "cisla", ne nase preference. (sam to take nevyuzivam, ale kdyz je po tom poptavka...)

Prioritne bych se ale zameril na nejakou ceskou platebni branu. Nyni sbiram zajemce...

**Comment by landsman** [comment]
Máme nějaký sheet?

**Comment by radimklaska** [comment]
Nene, zatim to pocitam jen v hlave :-) ono to totiz nejsou zadne velke pocty...

**Comment by landsman** [comment]
No, že by to bylo transparentnější, veřejné :)

**Comment by radimklaska** [comment]
"sheet" jsem pridal do issue :-)

**Comment by iBobik** [comment]
K těm platbám: Pokud to nemá nějaké nevýhody (ještě jsem to nedělal), tak se přimlouvám za integraci s [Payment](https://www.drupal.org/project/payment), ne s přímou integrací s Commerce. Na stránce Payment třeba ale nevidím nic o reccurring payments ("uložení" karty), což může být nevýhoda.

**Comment by secmi** [comment]
@iBobik Ono to bude asi tím, že reccurring payments/uložení karty jsou spíše záležitostí platební brány. Opravdu bych nikomu nepřál procházet certifikací na to, abych mohl ve svém SW ukládat čísla karet :(

**Comment by iBobik** [comment]
@secmi A proto se to dělá tak, že pokud uživatel s uložením na bráně souhlasí, tak eshopu předá klíč, kterým si může eshop provádět další platby už bez potvrzování platby zákazníkem.

V Commerce 1 to dělal [tento modul](https://www.drupal.org/project/commerce_recurring), v D8 je to součástí Commerce 2.

**Comment by secmi** [comment]
@iBobik Jasně, promiň asi jsem Tě špatně pochopil a čekal jsem, že chceš uložení karty přímo u prodejce... :(

**Comment by martin-klima** [comment]
Začal bych jedním nejžádanějším modulem, a to je GoPay. 
Návrh financování:
1. Příspěvek od GoPay
2. Příspěvek od potenciálních koncových klientů
3. Příspěvek od firem, které by to rádi pak prodali jako součást řešení (každý se zeptá ve své firmě)
4. Příspěvek od site builderů a vývojářů, kteří by to rádi pak prodali, ale nebudou to vyvíjet
5. Příspěvek od vývojářů, kteří to budou vyvíjet, protože ví, že to pak prodají za víc v nějakém řešení
6. Dobrovolný příspěvek všech ostatních (z komunity i mimo ní)
7. Příspěvek Asociace

Ideální by bylo, kdyby vývojáři pracovali na modulu "z nadbytku", tedy maximálně za uhrazené životní náklady bez komerčního zisku. Pro dobré jméno, dobrý pocit a pro budoucí rozvoj.

**Comment by iBobik** [comment]
Co tedy pro Gopay založit své issue, kde už budeme řešit podrobnosti?

**Comment by landsman** [comment]
Tohle by mohlo být taky easy portnout na D8: https://www.drupal.org/project/adminer, 
český vynález :)

**Comment by miloskroulik** [comment]
@landsman Není to trochu zbytečné, když ten modul nenabízí nic zásadně nového?
